src/rebuild/srv.c

```diff
@@ -570,14 +570,17 @@ rebuild_leader_status_check(struct ds_pool *pool, uint32_t map_ver,
 			    struct rebuild_global_pool_tracker *rgt)
 {
 	double		last_print = 0;
-	double		last_query = 0;
 	unsigned int	total;
 	int		rc;
 
 	rc = crt_group_size(pool->sp_group, &total);
 	if (rc)
 		return;
 
+	rgt->rgt_ult = dss_sleep_ult_create();
+	if (rgt->rgt_ult == NULL)
+		return;
```

Comment:
when it get NULL should it return ENOMEM?

Reply:
ah, this is void. probably I should output a warning here.